onnx_tf/frontend.py

```diff
@@ -243,18 +237,16 @@ def tensorflow_graph_to_onnx_graph(cls,
           # Remove tensorflow-specific attrs that are not
           # needed/allowed in ONNX.
           attr = cls.DEFAULT_TF_ATTR_PER_OP.get(node.op, {})
-          attr.update(
-              dict(
-                  filter(lambda pair: pair[0] not in TF_ATTR_TO_REMOVE,
-                         node.attr.items())))
-          node.attr = attr
-          node_output = node.name
+          filtered_attr = dict(
```

Comment:
Should avoid creating side effects whenever possible. 
Plus, later node translation can make use of previous nodes' attribute as hints.

Another desirable outcome is that, since now we don't overwrite attributes, we don't need to have a dictionary for a specific attribute (like output_shapes).